aws-cloudformation/aws-cloudformation-resource-providers-codeartifact
# Init Resource Handlers for AWS::CodeArtifact::Repository

Basic implementation of Create/Read/Delete handlers.


**Description of changes:**
Template code from `cfn-init` and implemented Create/Delete/Read handlers. 

**Testing**
Did `cfn-submit` on my own account, created a stack with a repository/domain template and deleted it.

Some TODOs:
+ update/list handlers
+ Add upstream, external connection, and policy handling for create/update handlers and to the resource schema


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
